frontend/src/pages/Worksheet.jsx

```diff
@@ -30,6 +30,35 @@ const StyledCalendarContainer = styled(SurfaceComponent)`
     color ${({ theme }) => theme.trans_dur};
 `;
 
+function WorksheetStats() {
+  const { courses, hidden_courses, cur_season } = useWorksheet();
+  const { courseCnt, credits, workload, rating } = courses.reduce(
```

Comment:
I think there may be a more efficient way to do this, let me make some changes but lmk what you think